app/helpers/injection_helper.rb

```diff
@@ -164,7 +165,6 @@ def default_enterprise_query
   end
 
   def enterprise_injection_data
-    @enterprise_injection_data ||= OpenFoodNetwork::EnterpriseInjectionData.new
-    { data: @enterprise_injection_data }
+    { data: OpenFoodNetwork::EnterpriseInjectionData.new }
```

Comment:
This is changing logic. The object was cached before which is important for performance in this case. Caching in helpers isn't great but I'm not sure how easy it is to resolve. We would need to look at all the places where the helpers are called. But really, injecting JSON data into the page is only used by AngularJS which we are phasing out. So keeping it as todo is probably best.